app/templates/reports/manage_report.html

```diff
@@ -0,0 +1,101 @@
+        <tr><td>Report ID</td><td>{{ report.id }}</td></tr>
+        <tr><td>Vehicle ID</td><td>{{ report.vehicle_id }}</td></tr>
+        <tr><td>License Plate</td><td>{{ report.license_plate }}</td></tr>
+        <tr><td>Bus Number</td><td>{{ report.bus_number }}</td></tr>
+        <tr><td>LED Screen Number</td><td>{{ report.led_screen_number }}</td></tr>
+        <tr><td>Location</td><td>{{ report.location }}</td></tr>
+        <tr><td>Date</td><td>{{ report.date }}</td></tr>
+        <tr><td>Duration</td><td>{{ report.duration}}</td></tr>
+        <tr><td>Agency</td><td>{{ report.agency.name }}</td></tr>
+        <tr><td>Picture</td><td><img src="{{ report.picture_url }}" alt="report picture" width="200"</td></tr>
+        <!-- TODO add width attr to styles -->
+        <tr><td>Description</td><td>{{ report.description }}</td></tr>
+    </table>
+{% endmacro %}
+
+{% block content %}
+    <div class="ui stackable centered grid container">
+        <div class="twelve wide column">
+            <a class="ui basic compact button" href="{{ url_for('reports.view_reports') }}"> <!-- TODO my reports -->
+                <i class="caret left icon"></i>
+                Back to all reports
+            </a>
+            <h2 class="ui header">
+                {{ '%s\'s report on %s' % (report.user.full_name(), report.agency.name) }}
+                <div class="sub header">View and manage this report.</div>
+            </h2>
+        </div>
+        <div class="stretched divided very relaxed row">
+            <div class="four wide column">
+                {{ navigation(endpoints) }}
+            </div>
+            <div class="eight wide column">
+                {% if request.endpoint == deletion_endpoint %}
+                    <h3 class="ui red block header">
+                        <i class="warning circle icon"></i>
+                        <div class="content">
+                            This action is permanent
+                            <div class="sub header">
+                                Deleting a report is not a reversible change. Any information associated
+                                with this report will be removed, and cannot be recovered.
+                            </div>
+                        </div>
+                    </h3>
+
+                    <div class="ui form">
+                        <div class="inline field">
+                            <div class="ui deletion checkbox">
+                                <input type="checkbox" tabindex="0" class="hidden">
+                                <label>I understand that this action cannot be undone.</label>
+                            </div>
+                        </div>
+                        <a class="ui disabled negative deletion button">
+                            Delete this report
+                        </a>
+                    </div>
+                    </div>
+                {% elif form %}
+                    {{ f.render_form(form) }}
```

Comment:
hmm i think supporting file upload would be pretty doable. we would have to have the functionality either way, right? i feel like it would actually be easier to support it bc we would already have it for creating a new report anyway